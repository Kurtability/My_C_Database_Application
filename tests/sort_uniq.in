sort a
uniq a
set a 2 1 1 1
get a
sort a
get a
uniq a
get a
push a 1 1 1
get a
uniq a
get a
bye
