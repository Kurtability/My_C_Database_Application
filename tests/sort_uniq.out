> sort a
no such key

> uniq a
no such key

> set a 2 1 1 1
ok

> get a
[2 1 1 1]

> sort a
ok

> get a
[1 1 1 2]

> uniq a
ok

> get a
[1]

> push a 1 1 1
ok

> get a
[1 1 1 1]

> uniq a
ok

> get a
[1]

> bye
bye
